src/foam/core/Validation.js

```diff
@@ -141,6 +141,15 @@ foam.CLASS({
             errorString: `Please enter a ${this.label} with at most ${this.maxLength} character${this.maxLength>1?'s':''}`
           });
         }
+        if ( this.required && ! foam.Number.isInstance(this.maxLength) && ! foam.Number.isInstance(this.minLength) ) {
```

Comment:
What if `maxLength: 10` and `required: true` are set, but `minLength` isn't? Then the `required` predicate won't apply, even though we want it to. So I think this condition needs to be updated to:

```suggestion
        if ( this.required && ! foam.Number.isInstance(this.minLength) ) {
```

Sorry I didn't catch this the first time.